lib/handlers/compile.js

```diff
@@ -235,6 +235,11 @@ class CompileHandler {
             _.each((req.query.removeFilters || "").split(","), filter => {
                 if (filter) delete filters[filter];
             });
+            // Ask for asm not to be returned
+            if (req.query.skipAsm === "true") {
```

Comment:
does the query not parse this as a `true` value? I forget how all this works :D

Reply:
The stuff that comes from express is returned as a string https://expressjs.com/es/4x/api.html#req.query :)